middleware/swagger-validator.js

```diff
@@ -298,7 +298,11 @@ var wrapEnd = function (req, res, next) {
         mHelpers.debugError(err, debug);
       }
 
-      return next(err);
+      if (responseValidatorHook) {
```

Comment:
I read up on Express middleware error handling, and I see that if you want to suppress an error and continue processing the request, you can't call `next(err)`, so I understand why this hook is justified (and how Express is limited).